docker-compose.yml

```diff
@@ -49,6 +49,11 @@ services:
     command: ["./container_init.sh"]
     environment:
       - DEPLOYMENT_STAGE=dev
```

Comment:
Out of scope question for @jgadling : since we don't have `dev` anymore, given the new deployment flow of `main` automatically deploys to `staging`, should we change `DEPLOYMENT_STAGE` to something else? 

Thank you!

Reply:
I think we've got a lot of cleanup to do for the config system in general, but we should probably tackle that separately. poking @mbarrien for visibility.

Reply:
That makes sense 👍 ! Thanks @jgadling 😄